Pod/Classes/VOKCoreDataManager.m

```diff
@@ -594,18 +619,22 @@ - (NSURL *)applicationLibraryDirectory
 
 - (void)resetCoreData
 {
-    NSArray *stores = [[self persistentStoreCoordinator] persistentStores];
+    if (_persistentStoreCoordinator) {
```

Comment:
Ahhh!  In that case, changing the deleted line 597 above to
```objc
NSArray *stores = [_persistentStoreCoordinator persistentStores];
```
should be sufficient to fix it (without needing the rest of the changes).  I'd also suggest adding a comment explaining why the direct ivar access is necessary.